tests/system/providers/amazon/aws/example_eks_with_nodegroups.py

```diff
@@ -132,8 +132,8 @@ def delete_launch_template(template_name: str):
         cmds=["sh", "-c", "echo Test Airflow; date"],
         labels={"demo": "hello_world"},
         get_logs=True,
-        # Delete the pod when it reaches its final state, or the execution is interrupted.
-        is_delete_operator_pod=True,
+        # Keep the pod alive, so we can describe it in case of trouble. It's deleted with the cluster anyway.
```

Comment:
It seems it is specific to system tests and not worth to be in the docs, it might confuse users. I would move it outside of the doc tag

Reply:
yeah good point, I'll do that for the other tests as well

Reply:
that's actually the only test where this is part of the doc example :)